src/Templates/Task/Event.java

```diff
@@ -0,0 +1,38 @@
+package Templates.Task;
+
+import Templates.BaseDate;
+
+public class Event extends Task {
+    private BaseDate startDateTime = null;
+    private BaseDate endDateTime = null;
+    public static String keyword1 = "/from";
+    public static String keyword2 = "/to";
```

Comment:
Perhaps a more descriptive name in all caps and underscore separation for these constants, for example KEYWORD_FROM and KEYWORD_TO?